diff --git a/android/build.gradle b/android/build.gradle
index 3d579f015938ab7cc95b400c3fdc67f9294062d7..1e79862ea608b654baae62f3b48cf053c35274a4 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -35,6 +35,7 @@ def isNewArchitectureEnabled() {
 }
 
 apply plugin: "com.android.library"
+apply plugin: "org.jetbrains.kotlin.android"
 
 // Remove per: https://github.com/callstack/react-native-builder-bob/issues/774#issuecomment-2674585468
 /*if (isNewArchitectureEnabled()) {
@@ -109,6 +110,6 @@ dependencies {
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation 'com.google.android.recaptcha:recaptcha:18.9.2'
+  implementation 'com.google.android.recaptcha:recaptcha:18.8.0'
 }
 
